### DISCUSSION TITLE
为什么现在设置主线图，进图的时候卡在选择队伍界面，困难图也是一样的

### DISCUSSION THREAD

**Discussion by Mashiro12138** [LmeSzinc/AzurLaneAutoScript#3222] [General]
为什么现在设置主线图，进图的时候卡在选择队伍界面，困难图也是一样的，然后会报错
Task `Main` failed 3 or more times.                        
CRITICAL 11:03:44.197 │ Possible reason #1: You haven't used it correctly. Please  
         read the help text of the options.                                        
CRITICAL 11:03:44.198 │ Possible reason #2: There is a problem with this task.     
         Please contact developers or try to fix it yourself.                      
CRITICAL 11:03:44.199 │ Request human takeover                                     
INFO     11:03:44.201 │ No provider specified, skip sending

**Comment by Mashiro12138** [comment]
以前都是可以正常使用的，然后今天没用重新进行使用报这个错了

**Reply by KKLT1221** [reply]
一样的，上周游戏更新后就一直卡到现在了，中间软件更新了两三次也不行

**Reply by KKLT1221** [reply]
重新下载安装包安装完解决了，和更新前一样流畅

**Comment by LmeSzinc** [comment]
#3205
